File: apps/web/src/types/city.ts
export type City = {
  id: number;
  city_code: string;
  city_name: string;
  state_name: string;
  zone: string | null;
  tier: string | null;
  demand_index: number | null;
  affordability_index: number | null;
  macro_growth_index: number | null;
  is_active: boolean;
  created_at: string;
  updated_at: string;
};

File: apps/api/app/schemas/city.py
from datetime import datetime

from pydantic import BaseModel, ConfigDict


class CityResponse(BaseModel):
    id: int
    city_code: str
    city_name: str
    state_name: str
    zone: str | None
    tier: str | None
    demand_index: float | None
    affordability_index: float | None
    macro_growth_index: float | None
    is_active: bool
    created_at: datetime
    updated_at: datetime

    model_config = ConfigDict(from_attributes=True)